### DISCUSSION TITLE
Add Obsidian DataView support

### DISCUSSION THREAD

**Discussion by GlibMartynenko** [GooRoo/mkdocs-obsidian-bridge#7] [Ideas]
Hi, 
Thank you for your work.
It would be nice to include Obsidian DataView functionality in future releases.
https://github.com/blacksmithgu/obsidian-dataview

**Comment by GooRoo** [comment]
Hi @GlibMartynenko,

That would be a nice feature indeed! However, considering how slow I am developing this plugin, I can't even give any reasonable estimation on when to expect this. 

Thank you for your idea anyway. I'll add it to the backlog.
